.github/pull_request_template.md

```diff
@@ -1,9 +1,15 @@
-###### Description
+##### Description
 
 Fill in your description here.
 
 ---
 
+##### Checklist
+
+Remove items which don't apply to your PR.
+
+- [ ] Changelog updated
+
```

Comment:
piggybacking on this PR 🤔

Reply:
I can submit this change separately if you want 🙄

Reply:
Better practice IMHO

Reply:
Allright, see https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1922